src/main/java/io/vertx/core/impl/ContextImpl.java

```diff
@@ -279,5 +279,11 @@ private void setTCCL() {
     Thread.currentThread().setContextClassLoader(tccl);
   }
 
-
+  public int getInstanceCount(){
+    if(deployment == null || deployment.deploymentOptions() == null) {
+      // implicitly there is the current instance of the verticle running always
```

Comment:
If deployment == null then it should return zero as it's possible to have a context without a verticle (e.g. in the embedded case)